src/lesson26/js/main.js

```diff
@@ -1,83 +1,209 @@
 import * as validation from './modules/validation.js';
 
-const username = document.getElementById('username');
+const username = document.getElementById('js-username');
+const email = document.getElementById('js-email');
+const password = document.getElementById('js-password');
+const confirmPassword = document.getElementById('js-confirmPassword');
 
 username.focus();
 
-const email = document.getElementById('email');
-const password = document.getElementById('password');
-const confirmPassword = document.getElementById('confirmPassword');
-
 let stateOfItems = [
-  { item: username, empty: true, isValid: false },
-  { item: email, empty: true, isValid: false },
-  { item: password, empty: true, isValid: false },
-  { item: confirmPassword, empty: true, isValid: false }
+  {
+    item: username,
+    name: 'username',
+    isEmpty: true,
+    isValid: false,
+    errorMessage: 'none',
+    theOtherErrorMessage: 'none'
+  },
+  {
+    item: email,
+    name: 'email',
+    isEmpty: true,
+    isValid: false,
+    errorMessage: 'none',
+    theOtherErrorMessage: 'none'
+  },
+  {
+    item: password,
+    name: 'password',
+    isEmpty: true,
+    isValid: false,
+    errorMessage: 'none',
+    theOtherErrorMessage: 'none'
+  },
+  {
+    item: confirmPassword,
+    name: 'confirmPassword',
+    isEmpty: true,
+    isValid: false,
+    errorMessage: 'none',
+    theOtherErrorMessage: 'none'
+  }
```

Comment:
単に初期なのでしたら差を分かりやすくすると読みやすいです

```suggestion
const baseStateOfItems = {
    isEmpty: true,
    isValid: false,
    errorMessage: 'none',
    theOtherErrorMessage: 'none'
}
.
.
.
  {
    item: username,
    name: 'username',
    ...baseStateOfItems
  },
  {
    item: email,
    name: 'email',
    isEmpty: true,
    isValid: false,
    errorMessage: 'none',
    theOtherErrorMessage: 'none'
  },
  {
    item: password,
    name: 'password',
    ...baseStateOfItems
   },
  {
    item: confirmPassword,
    name: 'confirmPassword',
   ...baseStateOfItems
  }
```

Reply:
Thank you ken-san:)
Thanks to your advice, my code is readable.
cac28afd5761bf7424a6d9ff10e5a615d7b2dbe7